cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2367,7 +2367,7 @@ search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-31m@m
 | `cs_current_week_login_count_o365` \
 | `cs_country_login_percentage` \
 | fillnull Country, Region, City value="-" \
-| eval cyences_severity = case(percentage_login_from_country==0, "high", user_type=="Admin" or user_type=="DcAdmin", "medium", true(), "low") \
+| eval cyences_severity = case(percentage_login_from_country==0, "critical", user_type=="Admin" or user_type=="DcAdmin", "high", true(), "medium") \
```

Comment:
@VatsalJagani Should we discuss this once with Usama in meeting? Last time we had to change from critical to high

Reply:
We can just inform today.